docs/modules/rust-guide/pages/multiply-dependency.adoc

```diff
@@ -248,7 +248,7 @@ service : {
 
 == Start the local network
 
-Before you can build the project, you need to connect to the {IC} network either running locally in your development environment or running remotely on a sub-network that you can access.
+Before you can build the project, you need to connect to the {IC} network either running locally in your development environment or the decentralized {IC} mainnet.
```

Comment:
I think I would suggest referring to "the decentralized IC mainnet" as "the IC". And maybe one connects to "a replica" instead of "the IC network"? Just my 2c.

Reply:
The goal as I understand it is to connect our special terminology to the terminology used in the wider industry.  To that extent, I have used the more common terminology (decentralized blockchain mainnet) and linked it to our terminology (IC/subnet) early in each page/document.  There are some things like "replica" which are so internal that I don't even think that they have a place in external documentation.  Given that "replica" literally refers to one of what will be a collection of processes I can't say that I have a clear idea of what a "replica" is vis-a-vis say a "node".  If it was me I would not use that term.

Reply:
updated with respect to discussion in slack: "Internet Computer blockchain".